packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -201,6 +192,11 @@ export class SlackService {
                     }
                 }
             } catch (e) {
+                if (this.lightdashConfig.mode === LightdashMode.PR)
```

Comment:
if it fails, I can get an error on a thread (only on PR, which is harder to debug) 

![image](https://user-images.githubusercontent.com/1983672/220349472-79d7a182-d45e-4d25-a7e3-c1edf7ecbadc.png)